src/store/epics/auctions.ts

```diff
@@ -246,7 +246,7 @@ const startAuctionEpic: RootEpic = (
               const transaction = new Builders.NFTBidBuilder()
                   .NFTBidAsset({
                       auctionId: auctionId,
-                      bidAmount: Utils.BigNumber.make(bidAmount),
+                      bidAmount: Utils.BigNumber.make(bidAmount * 10 ** 8),
```

Comment:
This constant  (10 ** 8) has been used in many places, can we put it to some constant (please fix in the whole project)?

Reply:
Moved to .env variable